Adding `export OMPI_MCA_osc="^ucx"` to the `test` step helped. I will add it to this patch.

For completion: the `output` file of the failing `step-76` is:
```
Running with 1 MPI processes
[b46f2dfe6e7c:06703] ../../../../../../ompi/mca/osc/ucx/osc_ucx_component.c:375  Error: OSC UCX component priority set inside component query failed 
 
Number of degrees of freedom: 25600 ( = 4 [vars] x 400 [cells] x 16 [dofs/cell/var] )
[b46f2dfe6e7c:06703] ../../../../../../ompi/mca/osc/ucx/osc_ucx_component.c:375  Error: OSC UCX component priority set inside component query failed 
 
[b46f2dfe6e7c:06703] ../../../../../../ompi/mca/osc/ucx/osc_ucx_component.c:375  Error: OSC UCX component priority set inside component query failed 
 
Time step size: 0.00127369, minimal h: 0.015, initial transport scaling: 0.00882437

Time:       0, dt:   0.0013, norm rho:  3.914e-16, rho * u:  1.431e-16, energy: 8.334e-16
Time:  0.0502, dt:   0.0011, norm rho:    0.02058, rho * u:    0.03785, energy:   0.08689
Time:   0.101, dt:    0.001, norm rho:    0.02186, rho * u:    0.04494, energy:   0.08829
Time:   0.151, dt:  0.00099, norm rho:    0.02239, rho * u:    0.04567, energy:   0.08858
Time:   0.201, dt:    0.001, norm rho:    0.02036, rho * u:    0.04337, energy:   0.08116
Time:    0.25, dt:    0.001, norm rho:    0.01672, rho * u:    0.04178, energy:    0.0676
Time:   0.301, dt:    0.001, norm rho:    0.01628, rho * u:    0.03983, energy:   0.06479
Time:    0.35, dt:    0.001, norm rho:    0.01754, rho * u:    0.04235, energy:   0.06954
Time:   0.401, dt:    0.001, norm rho:    0.01577, rho * u:    0.04472, energy:   0.06277
Time:   0.451, dt:    0.001, norm rho:    0.01505, rho * u:    0.04327, energy:   0.06071
Time:   0.501, dt:    0.001, norm rho:    0.01548, rho * u:    0.04165, energy:   0.06236
```

Might be related to https://github.com/open-mpi/ompi/issues/12517, https://github.com/open-mpi/ompi/issues/12892, https://github.com/open-mpi/ompi/issues/14086.

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -53,6 +53,8 @@ jobs:
             os: ubuntu-22.04
           - ubuntu_version: noble
             os: ubuntu-24.04
+          - ubuntu_version: resolute
+            os: ubuntu-26.04
 
     steps:
     - uses: actions/checkout@v7
@@ -104,7 +106,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ubuntu_version: [jammy, noble]
+        ubuntu_version: [jammy, noble, resolute]
         os: [ubuntu-24.04, ubuntu-24.04-arm]
         include:
           - os: ubuntu-24.04
@@ -158,6 +160,10 @@ jobs:
         # messaging module was unable to find any relevant network
         # interfaces."
         export OMPI_MCA_btl_base_warn_component_unused='0'
+        # Disable UCX OSC to prevent errors of the kind:
+        # "../ompi/mca/osc/ucx/osc_ucx_component.c:375  Error: OSC UCX
+        # component priority set inside component query failed"
+        export OMPI_MCA_osc='^ucx'
 
         cd build
         make \
@@ -187,7 +193,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ubuntu_version: [jammy, noble]
+        ubuntu_version: [jammy, noble, resolute]
 
     container:
       image: dealii/dependencies:${{ matrix.ubuntu_version }}-v9.7.1

